src/auth/choicewindow.cpp

```diff
@@ -0,0 +1,89 @@
+#include "choicewindow.h"
+#include "downloaderWrapper.h"
+
+#include <QLabel>
+#include <QListWidget>
+#include <QVBoxLayout>
+#include <QDialogButtonBox>
+#include <QAbstractButton>
+#include <QPushButton>
+#include <QDebug>
+#include <QApplication>
+#include <QDesktopWidget>
+
+ChoiceWindow::ChoiceWindow(QStringList &usersFilesList, QWidget *pParent)
```

Comment:
you don't change `QStringList &usersFilesList`
initialize it as **const**

Reply:
changed